pkg/dashboards/kubernetes/kubelet/kubelet.go

```diff
@@ -0,0 +1,158 @@
+	)
+}
+
+func withPodStartRateAndDuration(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Pod Start Rate and Duration",
+		panelgroup.PanelsPerLine(2),
+		panels.PodStartRate(datasource, labelMatcher),
+		panels.PodStartDuration(datasource, labelMatcher),
+	)
+}
+
+func withStorageOperationsAndErrors(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Storage Operations Rate and Errors",
+		panelgroup.PanelsPerLine(2),
+		panels.StorageOperationRate(datasource, labelMatcher),
+		panels.StorageOperationErrorRate(datasource, labelMatcher),
+	)
+}
+
+func withStorageOperationsQuantile(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Storage Operation Duration 99th quantile",
+		panelgroup.PanelsPerLine(1),
+		panels.StorageOperationDuration(datasource, labelMatcher),
+	)
+}
+
+func withCgroupManager(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Cgroup manager",
+		panelgroup.PanelsPerLine(2),
+		panels.CgroupManagerOperationRate(datasource, labelMatcher),
+		panels.CgroupManagerQuantile(datasource, labelMatcher),
+	)
+}
+
+func withPLEGRelist(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("PLEG relist",
+		panelgroup.PanelsPerLine(2),
+		panels.PLEGRelistRate(datasource, labelMatcher),
+		panels.PLEGRelistInterval(datasource, labelMatcher),
+	)
+}
+
+func withPLEGRelistDuration(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("PLEG relist duration",
+		panelgroup.PanelsPerLine(1),
+		panels.PLEGRelistDuration(datasource, labelMatcher),
+	)
+}
+
+func withRPCRate(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("RPC rate",
+		panelgroup.PanelsPerLine(1),
+		panels.RPCRate(datasource, labelMatcher),
+	)
+}
+
+func withRequestDurationQuantile(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Request duration 99th quantile",
+		panelgroup.PanelsPerLine(1),
+		panels.RequestDurationQuantile(datasource, labelMatcher),
+	)
+}
+
+func withKubeletResources(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Memory and CPU Usage",
+		panelgroup.PanelsPerLine(2),
+		panels.KubeletMemory(datasource, labelMatcher),
+		panels.KubeletCPU(datasource, labelMatcher),
+	)
+}
+func withKubeletGoRoutines(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Goroutines",
+		panelgroup.PanelsPerLine(1),
+		panels.KubeletGoRoutines(datasource, labelMatcher),
+	)
+}
+
+func BuildKubeletMixin(project string, datasource string, clusterLabelName string) dashboards.DashboardResult {
```

Comment:
Maybe we can follow `BuildXOverview` pattern here?
Mixin usually suggests dashboards + alerts + rules in jsonnet.

Reply:
Thanks makes total sense 🙇‍♀️